testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -1139,7 +1139,7 @@ impl RelayerThread {
             key_block_ptr: u32::try_from(key.block_height)
                 .expect("FATAL: burn block height exceeded u32"),
             key_vtxindex: u16::try_from(key.op_vtxindex).expect("FATAL: vtxindex exceeded u16"),
-            memo: vec![STACKS_EPOCH_3_1_MARKER],
+            memo: vec![STACKS_EPOCH_3_2_MARKER],
```

Comment:
I think we should leave this as 3.1 for the moment.

If this merges to develop and there's another release before 3.2, anyone running that version (i.e., `3.1.0.0.14`) would make commits with 3.2 marker even though their nodes aren't capable of upgrading yet